src/styles/sidebar/components/Buttons.scss

```diff
@@ -0,0 +1,68 @@
+@use "sass:map";
+@use "../../variables" as var;
+@use '../../shared/components/Buttons' as buttons;
+
+// An icon-only button with tighter padding (e.g. TopBar icons)
+.CompactIconButton {
+  // Override larger touch-target min size
+  @include buttons.configure(
```

Comment:
Here's an example of configuring the `Buttons` module to allow for extending/overriding in a clearly-defined way.